src/components/job-overview/index.js

```diff
@@ -78,15 +77,14 @@ const JobOverview = (props) => {
           { label: smallJobName(jobName) },
         ]}
       />
-      <main>
+      <main className="grid grid--gap-x-large">
```

Comment:
grid gap large is enough in my opinion
```suggestion
      <main className="grid grid--gap-large">
```

Reply:
this class does not exist

Reply:
just create it in `src/style/objects.grid.css`

Reply:
added